src/api.ts

```diff
@@ -161,6 +165,8 @@ export function parseJsonConvertingSnakeToCamel(str: string): AuthenticationInfo
             this.lastActiveAt = value
         } else if (key === "legacy_user_id") {
             this.legacyUserId = value
+        } else if (key === "impersonator_user") {
```

Comment:
should be impersonator_user_id

Reply:
This is actually how the backend is sending it, need to keep it for back-compat, but I think impersoantor_user_id is a better thing to expose externally.